test/utils/objects.go

```diff
@@ -70,7 +70,7 @@ func PrepareTestObjects(ctx context.Context, c client.Client, namespacePrefix st
 	o.StateConfigMap = &corev1.ConfigMap{
 		ObjectMeta: metav1.ObjectMeta{Name: "tf-state", Namespace: o.Namespace},
 		Data: map[string]string{
-			StateKey: `some state`,
+			StateKey: `{"terraform_version":"0.13.7"}`,
```

Comment:
Would it be possible to add a test case for 0.12.x and to verify that expected `state replace-provider` cmds are called?

Reply:
Done